### DISCUSSION TITLE
Whistle work?

### DISCUSSION THREAD

**Discussion by Striker183** [InterurbanDev/MinecraftRailroadBlocks#48] [Q&A] [answered]
Hello there,
in the mod there is a whistle and an old whistle. But how do they work? So far I haven't managed to get the thing to work with any redstone signal.

Greetings,
Alex

**Comment by InterurbanDev** [comment, marked as answer]
The whistle sign and old whistle sign blocks have no functionality. In real life, these signs are placed along a railroad and would instruct a train engineer to sound their whistle or horn. These blocks work best in conjunction with a mod that adds whistles, such as the Create mod or Immersive Railroading, to instruct players to sound their whistles.
